tests/parser/tstrongspaces.nim

```diff
@@ -27,7 +44,7 @@ when true:
   let b = 66
   let c = 90
   let bar = 8000
-  if foo+4 * 4 == 8 and b&c | 9  ++
+  if foo+4 * 4 == 8  and  b&c | 9  ++
```

Comment:
This is unfortunate though. Does 'and' with single spaces still work for most other things like ``x+y == 8 and foo*8 <= 90``?